src/lwc/geModalPrompt/geModalPrompt.js

```diff
@@ -0,0 +1,40 @@
+import { LightningElement, api } from 'lwc';
+import { fireEvent } from 'c/pubsubNoPageRef';
+
+export default class geModalPrompt extends LightningElement {
+
+    @api severity;
+    @api title;
+    @api message;
+    @api buttonText;
+
+    handleCloseModal() {
+        fireEvent(this.pageRef, 'geModalCloseEvent', {});
+    }
+
+    get titleSectionComputedClass() {
+        let baseClass = ['slds-box', 'slds-theme_alert-texture', 'slds-box_extension'];
+
```

Comment:
I think this getter can be simplified to 
        ```
        if (isEmpty(this.severity)) {
            baseClass.push('slds-theme_default');
        }
        baseClass.push($`slds-theme_`+ this.severity);
        return baseClass.join(' ');
        ```

Reply:
@voduyemi I was thinking similarly, but that doesn't do as much to enforce the vocabulary as the individual else-if checks. But maybe there's a combination approach that avoids all the else-ifs but checks against the supported list of LWC classes!

Reply:
Very true, we can have an array of severity levels and just check to see if the string passed in matches anything in the list before we push to `baseClass`, if not we push the default theme.